pages/your_problem/storage.md

```diff
@@ -33,18 +33,17 @@ The timeline and purpose of the storage are also deciding factors for choosing a
 To find out what storage solutions are available to you and which one fits your needs, you can reach out to the IT-department of your host institution. Also, there are often national infrastructures available, for your domain or specialised on computing services, that offers storage solutions as well.
 
 For active data, it is important that it is easily accessible for everyone that needs to work with it. See recommendations on how to store and manage data during the course of your project below. For recommendations about storage after the active phase of a project, see recommendations further down.
-For sensitive data, consider the [Data Sensitivity guidelines](data_sensitivity).
+For sensitive data, consider the [Data Sensitivity guidelines](data_classification).
```

Comment:
Change to [Data **Classification** guidelines] instead of [Data **Sensitivity** guidelines]